### DISCUSSION TITLE
Questionaires Template - Only english?

### DISCUSSION THREAD

**Discussion by Bestpilot91** [globaleaks/globaleaks-whistleblowing-software#3519] [Q&A]
Hi,

So I did a channel and assigned a questionnaire to it. This questionnaire is built from the questionnaire library. I can't find the questions added there in the text customisation tab in the settings. Therefore unable to change these questions from English to another language when the user decides to surf using a different language. Help

**Comment by evilaliv3** [comment]
Hello!

Thank you for the question.

To translate the questionaires, just enable a new language and start configuring the platform by using that language.

For example if you operate in Italian, and you configure something you will be saving the configuration in Italian.
